tests/src/test/scala/cats/tests/RegressionTests.scala

```diff
@@ -85,7 +85,7 @@ class RegressionTests extends CatsSuite {
 
   test("#500: foldMap - traverse consistency") {
     assert(
-      List(1,2,3).traverseU(i => Const(List(i))).getConst == List(1,2,3).foldMap(List(_))
+      List(1,2,3).traverse(i => Const.of[List[Int]](List(i))).getConst == List(1,2,3).foldMap(List(_))
```

Comment:
@djspiewak it varies on the right. @edmundnoble any other suggestion in terms what to do with this?